Subject: [PATCH] EGL: call vendor eglCreateWindowSurface with NULL attrib_list

Old adreno (200) gpu EGL libraries fail to create window surface
when non-empty attrib list is passed, which cause fatal error
in system ui process. Passing NULL prevents the crash.

This change is intended only for compatibility with the old gpu libraries.

diff --git a/opengl/libs/EGL/eglApi.cpp b/opengl/libs/EGL/eglApi.cpp
index 94dfe6a9de..35eeb994bd 100644
--- a/opengl/libs/EGL/eglApi.cpp
+++ b/opengl/libs/EGL/eglApi.cpp
@@ -738,7 +738,7 @@ EGLSurface eglCreateWindowSurface(  EGLDisplay dpy, EGLConfig config,
         anw->setSwapInterval(anw, 1);
 
         EGLSurface surface = cnx->egl.eglCreateWindowSurface(
-                iDpy, config, window, attrib_list);
+                iDpy, config, window, NULL);
         if (surface != EGL_NO_SURFACE) {
             egl_surface_t* s =
                     new egl_surface_t(dp.get(), config, window, surface, colorSpace, cnx);
